invenio_accounts/sessions.py

```diff
@@ -42,6 +45,27 @@
 _sessionstore = LocalProxy(lambda: flask.current_app.kvsession_store)
 
 
+def _get_user_country():
+    """Get user country."""
+    user_real_ip = flask.request.headers.get(
```

Comment:
You can assume that the real ip address is in ``request.remote_addr``. When the request is forwarded you need to use the WSGIProxyFix middleware.